### DISCUSSION TITLE
Chapter 9, Lesson 25 Compiling error, Reason: setup failed: panic: arithmetic underflow or overflow (0x11)]

### DISCUSSION THREAD

**Discussion by truth2012** [Cyfrin/foundry-full-course-cu#1335] [Q&A]
........
This the setUp function
 function setUp() external {
        DeployRaffle deployer = new DeployRaffle();
        (raffle, helperConfig) = deployer.run();
        (
            entranceFee,
            interval,
            vrfCoordinator,
            gasLane,
            subscriptionId,
            callbackGasLimit,
            link
        ) = helperConfig.activeNetwrokConfig();
        vm.deal(PLAYER, STARTING_USER_BALANCE);
    }
........

**Comment by AiULe** [comment]
Hello，I speculate `(...) = helperConfig.activeNetWorkConfig();`， this error is caused by inconsistency in the left and right sides of the equals sign

**Reply by AiULe** [reply]
You try to check

**Reply by truth2012** [reply]
Okay thanks a lot

**Reply by badar-chachar** [reply]
@truth2012 how did you resolve this error? i am getting the same error and i have double checked the return values equals to the variables.
